pkg/kubeapiserver/authorizer/modes/modes.go

```diff
@@ -36,6 +36,9 @@ const (
 // AuthorizationModeChoices is the list of supported authorization modes
 var AuthorizationModeChoices = []string{ModeAlwaysAllow, ModeAlwaysDeny, ModeABAC, ModeWebhook, ModeRBAC, ModeNode}
 
+// RepeatableAuthorizerTypes is the list of Authorizer that can be repeated in the Authorization Config
+var RepeatableAuthorizerTypes = []string{ModeWebhook}
```

Comment:
move this to point of use and avoid exporting a variable for this (will refactor a bit when the validation moves to k8s.io/apiserver)